frontend/src/libraries/language.ts

```diff
@@ -138,13 +152,14 @@ function findBestSupportedLocale(appLocales, browserLocales) {
         const existingMatch = matchedLocales[matchedPartialLocale];
 
         // Deduct a thousandth for being non-exact match.
+        const newMatchScore = 0.999 - index / walletLocales.length;
```

Comment:
That's a bug that has been discovered with types.
It still exists in a wallet, we should probably report it after merging that.

Reply:
@luixo you can create an issue on [wallet](https://github.com/near/near-wallet)